packages/common/src/lib.rs

```diff
@@ -1,7 +1,6 @@
-pub mod dto;
 mod error;
```

Comment:
```suggestion
pub mod dto;
mod error;
```

@Bonifaceebuka 
PR Testing failed because of this line is removed.